tests/Io/FileSystemTest.php

```diff
@@ -42,6 +42,130 @@ public function tearDown()
         $this->exec('rm -rf ' . $this->tmp);
     }
 
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error getting real path of invalid_path, check that the path exists
+     */
+    public function testGetRealPathWithInvalidPath()
+    {
+        FileSystem::getRealPath('invalid_path');
+    }
+
+    public function testGetRealPath()
+    {
+        $filePath = __DIR__ . '/../../composer.json';
+        $expectedRealPath = realpath($filePath);
+
+        $this->assertSame($expectedRealPath, FileSystem::getRealPath($filePath));
+    }
+
+    public function testWriteWithAppendFlag()
+    {
+        $this->assertSame(4, FileSystem::write('./temp_file', 'data', true));
```

Comment:
It would be good to check that this actually appends, by creating the file before, and finally reading its contents!